bin/test

```diff
@@ -16,4 +16,6 @@ node_modules/.bin/nyc \
   --statements  0 \
   --reporter    text \
   --reporter    html \
+  --extension   .mjs \
+  --extension   .js \
```

Comment:
Is the order in which extensions are specified significant?

Reply:
I don't think it is.

Reply:
In that case, :abc: please. :)

Reply:
:zap: